.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,122 @@
+name: Docker
```

Comment:
I'd prefer using buildah/podman tooling instead but since you already completed all of it, let's keep Docker here

Reply:
In principle we can replace this with podman equivalents. The only downside, is that I'm not sure how much maintenance are those specific github actions getting. 
Docker takes a good care of theirs afaik.